[test_badmicoctrl_1]:pr_test_with_helloworld

badmicroctrl opened the pull request:
hello world

Author: badmicroctrl@163.com


<!--
Thanks for opening a pull request!
If this is your first pull request you can find detailed information on how 
to contribute here:
  * [贡献规范](https://github.com/datawhalechina/whale-starry/blob/main/stl/README.md#4%E8%A7%84%E8%8C%83)

Rename your pr title:
    [Team_Task]: Your Custom Title.

-->

### What?
for paticipating the programme , submit a pr
<!--
- brief: 描述本次pr的内容简介
- author: 作者 
-->

### What changes are included in this PR?
include a helloworld code
<!--
当前pr包含了哪些文件变更?
-->

### Are these changes tested?
tested,result as follow:
![1687003909566](https://github.com/datawhalechina/whale-starry/assets/35834810/6fab89d3-ffbc-4795-94dc-641d497c1d27)

<!--
是否对编写的代码进行过测试？
或者
是否对学习的开源项目测试？
-->

Light-City commented:
Are these changes tested?
tested

this answer is so bad.

Light-City commented:
1.The directory structure is incorrect, you can refer to what others have done, or this document

https://github.com/datawhalechina/whale-starry/blob/main/stl/rules/pr.md

2.your pr title style is wrong.

badmicroctrl commented:
> 1.The directory structure is incorrect, you can refer to what others have done, or this document
> 
> https://github.com/datawhalechina/whale-starry/blob/main/stl/rules/pr.md
> I corrected the directory structure.
> 2.your pr title style is wrong.

modified

Light-City commented:
reject.

Light-City commented:
The directory structure is incorrect, you can refer to what others have done, or this document

https://github.com/datawhalechina/whale-starry/blob/main/stl/rules/pr.md

badmicroctrl commented:
> The directory structure is incorrect, you can refer to what others have done, or this document
> 
> https://github.com/datawhalechina/whale-starry/blob/main/stl/rules/pr.md

I modify the directory structure again, please check whether this time is ok , if ok I can enter the group...

martinx commented:
The structure is still incorrect, you can refer to the following :

```
-- 团队名称/   (目录，小写字母和下划线)
    README.md    (文件，团队介绍）
    --- 个人名称   (目录，小写字母和下划线)
       ---- src        （源文件目录）
            main.cpp   (Hello World 的演示代码)
       RADME.md    (文件，个人文档，记录等）

```

badmicroctrl commented:
> The structure is still incorrect, you can refer to the following :
> 
> ```
> -- 团队名称/   (目录，小写字母和下划线)
>     README.md    (文件，团队介绍）
>     --- 个人名称   (目录，小写字母和下划线)
>        ---- src        （源文件目录）
>             main.cpp   (Hello World 的演示代码)
>        RADME.md    (文件，个人文档，记录等）
> ```

I rectify the files according to your guidance, please check ,thank you

martinx commented on badmicroctrl_team/badmicroctrl/src/helloworld.cpp:
```diff
@@ -0,0 +1,10 @@
+#include <iostream>
+
+int main() {
+    std::cout << "Hello, World!" << std::endl;
```
Indent 2 spaces

martinx commented:
LGTM
